Adding individual contributor (#4510)
the-turing-way/the-turing-way: book/website/communication/open/curriculum-development.md

Once the program and its requirements (program learning outcomes) are known, the year of study provides an indication of the level of study. 
With this context set, you can move on to specifics about the course, such as the **course learning outcomes** and the level of competency expected by the end of the course. 

> These outcomes can be developed using **Bloom's Taxonomy** [Figure 1](#effective-learning-outcomes), where earlier courses on a program focus more on remembering and understanding information, while later courses move towards evaluating, analysing and creating new methods and

Reviewer: ```suggestion
These outcomes can be developed using [Bloom's Taxonomy](#effective-learning-outcomes), where earlier courses on a program focus more on remembering and understanding information, while later courses move towards evaluating, analysing and creating new methods and
```

  evidence of learning” and “Teaching and Learning (type of resources and materials)” ending with “Plan learning and teaching techniques” and “End”. On the right, Bloom’s Taxonomy is presented
  as a vertical list of six stages for designing learning outcomes.
---
_System for Developing Effective Learning Outcomes for Curriculum or Lessons._ Image created by Gule Saman building on John Biggs and Benjamin Bloom's work. Used under a CC-BY 4.0 license.
```

> The learning outcomes can be developed using the helpful list of verbs given in [Figure 2](#blooms-taxonomy-verbs). 

Reviewer: ```suggestion
The learning outcomes can be developed using the helpful list of verbs given in [](#blooms-taxonomy-verbs). 
```

||Gamification
||Field trips to labs and companies to observe theory in practice

:::

> Each of these sensory channels corresponds to specific teaching methods and techniques [Table 2](#table2). 

Reviewer: ```suggestion
Each of these sensory channels corresponds to specific teaching methods and techniques (see [](#table-learning-facilitation)). 
```
